firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClient.java

```diff
@@ -86,21 +87,25 @@
   private final long connectTimeoutInSeconds;
   private final long readTimeoutInSeconds;
 
+  private ConfigLogger configLogger;
```

Comment:
can this be `private final ConfigLogger`?

Reply:
Done.